### ISSUE TITLE
how does it handle cookies?

### ISSUE THREAD

**Issue by nicolai86** [matryer/silk#14]
Hey! 

Silk looks interesting! How does it handle dependent api requests, and cookie based authentication? 

Eg a user logs in, then CRUDs a todo?

**Comment by ajayep** [comment]
Hi ,

Silk is really cool. I too have the same issue on handling cookie based authentication with Silk

**Comment by matryer** [comment]
Great question. Right now it doesn't. How should it?

**Comment by matryer** [comment]
Gophers can use the runner package to do this easily enough I think. But I wonder if it's common enough a problem to solve in the docs?

**Comment by matryer** [comment]
For cookies, how about this:

```
## GET /protected-resource
  * Cookie: authtoken=value
  * Cookie: name=John
```

* It would support multiple lines, so you could add other cookies
* this would also be how the assertions looked too

**Comment by ajayep** [comment]
Cookie set by the server needs to be copied by the client for all subsequent requests. I believe this would involve some tweak in the runner package as well.

**Comment by nicolai86** [comment]
@matryer I like your markdown suggestion. This would be really great!

**Comment by matryer** [comment]
I think https://github.com/matryer/silk/issues/2 would address the sharing/reusing data, and cookies should be included too.

**Comment by matryer** [comment]
Can you guys please review this: https://github.com/matryer/silk/pull/27

It turns out, you can already set cookies using HTTP headers :) That was a nice surprise.

**Comment by nicolai86** [comment]
@matryer it works, but without #2 it's of no use to me since I need to store server send values and use them in subsequent requests.

**Comment by matryer** [comment]
Please try #2, now implemented in v0.4.0
